Containerfile.rag

```diff
@@ -0,0 +1,37 @@
+# vim: set filetype=dockerfile
+
+ARG OLS_VERSION=latest
+FROM quay.io/openshift/lightspeed-service-api:${OLS_VERSION}
+
+ARG RAG_INDEX="https://github.com/ilan-pinto/lightspeed-rag-documents/releases/latest/download/local.zip"
+ARG APP_ROOT=/app-root
+USER 0
+
+RUN microdnf install -y unzip wget \
+  && mkdir -p vector-db \
+	&& cd vector-db \
+  && wget $RAG_INDEX \
+	&& unzip local.zip \
+	&& mv local ocp-product-docs \
+	&& rm -f local.zip \
+	&& microdnf remove -y unzip wget
+
+RUN chown -R 1001:0 ${APP_ROOT} && \
+    chmod -R g+rx ${APP_ROOT}
+
+# activate virtualenv with workaround RHEL/CentOS 8+
```

Comment:
i don't think you should need this:
https://github.com/moby/moby/issues/33242#issuecomment-302075518

were you getting failures without it?

also i'm not even sure our base image should be using a venv, there should be no reason we can't just install our deps into the system directories, so we should revisit the OLS base image containerfile too(but not in this PR of course).  It's kind of the whole idea of docker images.

Reply:
this was copied from our existing container. Happy to remove it.

Reply:
if you feel up to exploring it, i'd be interested to see if we can remove it from the ols image containerfile also.